scripts/dist.ts

```diff
@@ -224,7 +219,15 @@ function main() {
       }
       // Start from the source even if dist is given
       if (filePath.endsWith(".dist")) {
-        filePath = filePath.substring(0, filePath.length - 5);
+        const candidateFilePath = filePath.substring(0, filePath.length - 5);
+
+        // Make sure this isn't an orphan dist file
```

Comment:
Maybe I should use a different word here? "Orphan catching" sounds truly Dickensian